### PR TITLE
docs: add whatsapp/sms channel in notification

### PR DESCRIPTION
**Pull request by Mangesh-Khairnar**
whatsapp:sms channel docs

feature link:https://github.com/frappe/frappe/pull/9623

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/setting-up/notifications.md [hunk 1]
```diff
@@ -162,6 +162,32 @@ If Email/Slack alerts and System Notifications both are required, the main chann
 
 <img class="screenshot" alt="Notifications Dropdown" src="{{docs_base_url}}/assets/img/setup/notifications/send-system-notification.png">
 
+## 4. WhatsApp
+In **Version 13** we have introduced an additional channel to send alerts - **WhatsApp**:
+<img class="screenshot" alt="Notifications WhatsApp Channel" src="{{docs_base_url}}/assets/img/setup/notifications/twilio-channel.png">
+
+If you prefer to have your notifications sent to a WhatsApp number, you can also choose the option "WhatsApp" in the channel options and select the appropriate Twilio Number. Twilio Numbers can be added to the Twilio settings. WhatsApp messages can only be sent to numbers which have country codes in them.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/setting-up/notifications.md:169]:
```suggestion
If you prefer to have your notifications sent to a WhatsApp number, you can also choose the option "WhatsApp" in the channel options and select the appropriate Twilio Number. Twilio Numbers can be added to Twilio settings in ERPNext. WhatsApp messages can only be sent to numbers which have country codes in them.
```

File: erpnext_documentation/www/docs/user/manual/en/setting-up/notifications.md [hunk 2]
```diff
@@ -162,6 +162,32 @@ If Email/Slack alerts and System Notifications both are required, the main chann
...
+
+<img class="screenshot" alt="Twilio Settings" src="{{docs_base_url}}/assets/img/setup/notifications/twilio-settings.png">
+
+### 2.1 Twilio Settings
+
+In order to configure the Twilio settings, you need to first obtain Twilio credentials from account settings of your Twilio Account. You can only add those phone numbers that have been activated in your Twilio Account with WhatsApp access.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/setting-up/notifications.md:175]:
```suggestion
In order to configure Twilio settings, you need to first obtain Twilio credentials from your Twilio Account's account settings. You can only add those phone numbers that have been activated in your Twilio Account with WhatsApp access.
```

File: erpnext_documentation/www/docs/user/manual/en/setting-up/notifications.md [hunk 3]
```diff
@@ -162,6 +162,32 @@ If Email/Slack alerts and System Notifications both are required, the main chann
...
+<img class="screenshot" alt="Twilio Credentials" src="{{docs_base_url}}/assets/img/setup/notifications/twilio-credentials.png">
+
+### 2.2 Message Format
+
+WhatsApp allows users to only send pre-approved templates to your users. Failure to do so might result in restrictions on you Twilio account.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/setting-up/notifications.md:180]: unclear, who is users and who approved the templates?

File: erpnext_documentation/www/docs/user/manual/en/setting-up/notifications.md [hunk 4]
```diff
@@ -162,6 +162,32 @@ If Email/Slack alerts and System Notifications both are required, the main chann
...
+<img class="screenshot" alt="Pre Approved Message Template" src="{{docs_base_url}}/assets/img/setup/notifications/twilio-pre-approved-message.png">
+
+
+## 5. SMS
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/setting-up/notifications.md:184]: afaik, SMS existed in notifications, it's not new?

**Review by prasadarr — CHANGES_REQUESTED** [review]

File: erpnext_documentation/www/docs/user/manual/en/setting-up/notifications.md (hunk 4 above)
Comment (Mangesh-Khairnar) [reply erpnext_documentation/www/docs/user/manual/en/setting-up/notifications.md:184]: No, this is a new channel

**Review by prasadarr — APPROVED** [review]

**Pull request merged**